lib/system.nim

```diff
@@ -674,6 +674,7 @@ when not defined(JS) and not defined(nimSeqsV2):
       len, reserved: int
       when defined(gogc):
         elemSize: int
+        elemAlign: int
```

Comment:
I doubt Go's layout rules for seqs care about ``elemAlign``.

Reply:
It is used here: https://github.com/krux02/Nim/blob/fc6965e0487aa328efaaef6a3830b3caec7a7e62/lib/system/mmdisp.nim#L331